src/commcare_cloud/commands/ansible/service.py

```diff
@@ -274,18 +281,21 @@ def _get_service_host_groups(self, process_pattern):
 class Nginx(AnsibleService):
     name = 'nginx'
     inventory_groups = ['proxy']
+    log_location = '/home/cchq/www/{env}/log/{env}_commcare-nginx_error.log'
```

Comment:
should this also have commcare-nginx_access.log?

Reply:
https://github.com/dimagi/commcare-cloud/pull/2033/commits/35b10dc59848436656d6bf84d13f5f54d9df9608